src/WebJobs.Script.WebHost/Controllers/ExtensionsController.cs

```diff
@@ -57,10 +57,10 @@ public async Task<IActionResult> Get()
 
         [HttpPost]
         [Route("admin/host/extensions")]
-        public Task<IActionResult> Post([FromBody]ExtensionPackageReferenceWithActions package) => InstallExtension(package);
+        public Task<IActionResult> Post([FromBody] ExtensionPackageReferenceWithActions package) => InstallExtension(package);
 
-        [HttpPut("{id}")]
-        [Route("admin/host/extensions")]
+        [HttpPut]
+        [Route("admin/host/extensions/{id}")]
```

Comment:
>{id} [](start = 38, length = 4)

This `id` is not even being used by `InstallExtension`. Okay to remove? `ExtensionPackageReferenceWithActions` has a separate `id` being fed as input and that is what is used by `InstallExtension`